src/app.ts

```diff
@@ -319,16 +339,16 @@ function appInit(gopt: AppOpt = {}): App {
 		return app.mouseDeltaPos.clone();
 	}
 
-	function isMouseClicked(): boolean {
```

Comment:
Should this default to left mouse button, or any mouse button like how `isKeyPressed()` without argument works?